resources/splunk_app.rb

```diff
@@ -69,25 +65,43 @@ def parse_meta_access(access)
     end
   end
 
+  # Provider exclusive methods
+
   action_class do
     include CernerSplunk::ProviderHelpers
 
+    def upgrade_app(strategy)
+      require 'fileutils'
```

Comment:
I feel like for readability, any requires that are just built into ruby and will happen on all platforms we should have outside. Is there a reason to delay evaluation of this require?